spec/controllers/idv/image_uploads_controller_spec.rb

```diff
@@ -219,6 +221,8 @@
             ocr_pii: nil,
             doc_type_supported: true,
             failed_image_fingerprints: { front: [], back: [] },
+            selfie_live: true,
+            selfie_quality_good: true,
```

Comment:
One thought can use parameterized shared examples like 
https://github.com/18F/identity-idp/blob/8cef39902ce5660daf56f2a382eb55ee0b6fb8de/spec/features/users/sign_in_spec.rb#L698-L698

For example, but it may not be necessary for simple checks.
```ruby
it_behaves_like 'with selfie result', selfie_live: true, selfie_quality_good: true
```

Reply:
That would simplify adding it everywhere. I'm wondering more if the fact that I have to change a lot of tests means I've made a mistake in the code (or misunderstood something).

Reply:
it's the nature to validate complicated data structure. Generally need to extract logic and use kind of validator, the shared example is the rspec way.

On the other hand, is it necessary to repetitively validate similar things over and over?